composer.json

```diff
@@ -44,6 +44,8 @@
             "dev-master": "2.0.x-dev"
         }
     },
+    "minimum-stability": "dev",
+    "prefer-stable": true,
```

Comment:
-1 for this change. I want tests to run against stable versions in most builds. Otherwise, there is nothing ensuring we can run without forcing dev deps to our users